doc/dev/installation.rst

```diff
@@ -88,50 +192,60 @@ installed and available on the path (depending on the platform):
 
 - The Python package manager ``pip`` must be version 22.3 or higher.
 
-They can be installed on **Debian/Ubuntu** via:
+They can be installed via:
 
-.. code-block:: console
 
-  sudo apt install clang lld ccache libomp-dev ninja-build make cmake
+.. tabs::
 
-.. Note::
+   .. group-tab:: Linux Debian/Ubuntu
 
-  If the CMake version available in your system is too old, you can also install up-to-date
-  versions of it via ``pip install cmake``.
+      .. code-block:: console
 
-On **macOS**, it is strongly recommended to install the official XCode Command Line Tools
-(for ``clang`` & ``make``). The remaining packages can then be installed via ``pip`` and ``brew``:
+        sudo apt install clang lld ccache libomp-dev ninja-build make cmake  # already in minimal guide
 
-.. code-block:: console
+      .. note::
 
-  pip install cmake ninja
-  brew install libomp
+        If the CMake version available in your system is too old, you can also install up-to-date
+        versions of it via ``pip install cmake``.
 
-All additional build and developer dependencies are managed via the repository's
-``requirements.txt`` and can be installed as follows:
 
-.. code-block:: console
+   .. group-tab:: macOS
 
-  pip install -r requirements.txt
+      On **macOS**, it is strongly recommended to install the official XCode Command Line Tools (for ``clang`` & ``make``). The remaining packages can then be installed via ``pip`` and ``brew``:
+
+      .. code-block:: console
+
+        xcode-select --install      # already in minimal guide
+        pip install cmake ninja     # already in minimal guide
+        brew install libomp         # already in minimal guide
```

Comment:
Not sure these comments are needed since they add a bit noise. Is there a situation were you think these markings would be very useful?

Reply:
I do think they are needed. For a new user who is unfamiliar with what commands to run, they won't have the explicit awareness that some commands were already run in the minimal, especially if they just copy paste the entire minimal guide without carefully reading what commands they just ran. I'd like some indicator that a command in the detailed section was already covered in minimal and so the user wouldn't re-run them.

Reply:
I agree that this kind of markings is a bit unclean, but I'm not sure what other markings could be used: rst codeblocks apparently do not support bold/italics/coloring/etc.

Reply:
I have removed them.